content/articles/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/index.md

```diff
@@ -0,0 +1,281 @@
+2. Symbolic math in the **live editor**(mode in MATLAB) lets you interactively update and display Symbolic math computations. Also, MATLAB code formatted text, equations, and images can be published as executable live scripts, PDFs, or HTML documents. While working with analytical problems, you can get suggestions for the next step. These suggestions help you insert and execute function calls or live editor tasks directly into live scripts.
+3. In addition to providing exact rational math, the Symbolic toolbox provides precision for higher or lower positions, allowing algorithms to run higher or lower than MATLAB's in-built double. It also includes units for working with physical quantities and performing dimensional analysis.
+
+![Units for working with physical quantities](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-one.jpeg)
+
+*This is an example of how this toolbox adds units for physical quantities*
+
+4. This toolbox is widely applied in many engineering and scientific application such as optimization, where symbolic expressions of exact gradient and Hessians improves accuracy and speed of optimization.
+5. In non-linear control design, the Symbolic toolbox improves recalculation speed at any operating point during execution. Furthermore, you can integrate symbolic results with MATLAB and Simulink applications by converting symbolic expressions into numeric MATLAB functions, Simulink, and Simscape blocks.
+
+![converting functions to simulink](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-two.jpeg.jpeg)
+
+*Sample of function converted to Simulink*
+
+Now, all these applications discussed above were to give you an insight into the wide application of this toolbox. However, not all of them are discussed here. Here, we will only major in using the toolbox to solve Fourier series problems.
+
+### How to use the symbolic toolbox
+This toolbox is enabled in MATLAB using the function `syms`. If you have an expression `x=2*a+b`, and you try to execute it in Matlab, you get an error message *undefined function or variable 'a'* as shown below:
+
+```MATLAB
+x = 2*a + b
+```
+Output:
+
+```MATLAB
+Undefined function or variable 'a'.
+```
+When using the `syms` function, the variable `x` is saved without the error message. The idea here is, when using the Symbolic toolbox, you first define the symbolic variables. Symbolic variables are the undefined variables in an equation. For our expression above, our symbols are `a` and `b`, so we first define these variables using the symbolic function `syms`.
+
+```MATLAB
+syms a b
+```
+After defining the symbols and rerunning the code above, our variables are stored in our workspace. We will then have:
+
+```MATLAB
+syms a b
+x = 2*a + b
+```
+Output:
+
+```MATLAB
+x =
+ 
+2*a + b
```

Comment:
```suggestion
x = 2*a + b
```

Reply:
@srishilesh these are the outputs and @munubi254 said they should be copied as they are. If you execute the commands in the command window, you get the outputs as I have given them out.

Reply:
@linetachieng  Got it.  Aren't these outputs from a bash terminal? If so, please modify the type of code snippet to bash

Reply:
> @linetachieng Got it. Aren't these outputs from a bash terminal? If so, please modify the type of code snippet to bash

@srishilesh, they are MATLAB outputs. I suggested that she copy pastes them as they are.